src/ucp/core/ucp_worker.c

```diff
@@ -300,6 +304,9 @@ ucp_worker_iface_error_handler(void *arg, uct_ep_h uct_ep, ucs_status_t status)
     if (ucp_ep_errh_iter != kh_end(&worker->ep_errh_hash)) {
         err_handler = kh_val(&worker->ep_errh_hash, ucp_ep_errh_iter);
         err_handler.cb(err_handler.arg, ucp_ep, status);
+    } else {
+        ucs_error("Error %s was not handled for ep %p",
```

Comment:
as i see it there are 2 options:
1. pass extra info to upper level
2. return status to lower level and let it print the error if unhandled
either way is good, as long as the error is printed only if unhandled, no more than once, and contains all relevant info.